assets/workflow/Immunobiology/Immunobiology.protocol 5.html

```diff
@@ -1,85 +1,123 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml">
-<head>
-<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-<title>Untitled Document</title>
-<style>
-img {
-    display: block;
-    margin-left: auto;
-    margin-right: auto;
-}
-</style>
-</head>
+<html>

```

Comment:
<img width="1743" alt="image" src="https://github.com/faryabiLab/hpap-public/assets/56408423/cb95680f-21a2-42c3-bd24-db78d8b67c82">
Please change CRLF here to LF for all these files

Reply:
in vscode setting for prettier:
<img width="438" alt="image" src="https://github.com/faryabiLab/hpap-public/assets/56408423/6e510659-84e3-4ff3-84ea-469bfac8cca3">

Reply:
Git it Thanks!